src/haddock/modules/scoring/emscoring/cns/scoring.cns

```diff
@@ -145,7 +156,6 @@ do (z = refz) (all)
 {* ===================== calculate complex internal energy *}
 
 evaluate ($kinter = 1.0)
-energy end
```

Comment:
Isn't this needed to finish the calculation?

Reply:
No - already included in `calc_free-ene.cns` - was redundant.